src/resources/views/crud/fields/relationship/entries.blade.php

```diff
@@ -25,6 +25,23 @@
     $pivotSelectorField['minimum_input_length'] = $pivotSelectorField['minimum_input_length'] ?? 2;
     $pivotSelectorField['delay'] = $pivotSelectorField['delay'] ?? 500;
     $pivotSelectorField['placeholder'] = $pivotSelectorField['placeholder'] ?? trans('backpack::crud.select_entry');
+
+    switch(gettype($field['reorder'])) {
+        case 'string' : {
+            $field['subfields'] = Arr::prepend($field['subfields'], [
+                'name' => $field['reorder'],
+                'type' => 'hidden',
+                'attributes' => [
+                    'data-reorder-input' => true
+                ]
+            ]);
+        }
+        break;
+        case 'array' : {
+            $field['subfields'] = Arr::prepend($field['subfields'], $field['reorder']);
+        }
+        break;
+    }
```

Comment:
Why not move this to `repeatable`? It's his feature, after all. 

The way I see it, if we do move this to `repeatable`, we open up the possibility for somebody to use this feature in `repeatable` with a custom saving process, without a relationship.